README.md

```diff
@@ -40,11 +40,25 @@ LEAP-Accelerate includes:
 
 `module load cmake/3.15.1 gcc/6.3.0 boost/1.66.0 casacore/3.1.2`
 
+`module unload gfortran/default`
+
+`module load isl/default`
+
+`export CUDA_HOME=/usr/local/cuda`
+
+`export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${CUDA_HOME}/lib64:${CUDA_HOME}/extras/CUPTI/lib64`
+
+`export PATH=$PATH:$CUDA_HOME/bin`
+
+`mkdir -p build && cd build`
+
+`cmake ../../ -DCASACORE_ROOT_DIR=/usr/local/casacore -DCUDA_HOST_COMPILER=g++ -DCUDA_TOOLKIT_ROOT_DIR="${CUDA_HOME}" -DCMAKE_BUILD_TYPE=Release`
+
```

Comment:
These instructions are missing the rapidjson dependency. I installed one locally under /home/rtobar/rapidjson and added `-DCMAKE_PREFIX_PATH=$HOME/local/rapidjson` to my `cmake` command line to get past the problem.

Here you also point to the casacore installation under `/usr/local/casacore`, which is different from that in the `casacore/3.1.2` module. I used `-DCASACORE_ROOT_DIR=$BLDR_CASACORE_BASE_PATH` instead (which points to the 3.1.2 installation), and that yielded a successful compilation. I also tried *not* loading `casacore/3.1.2` and using `-DCASACORE_ROOT_DIR=/usr/local/casacore`, but that indeed fails to link. This is due to the C++ ABI breaking change introduced in gcc 5.1: `/usr/local/casacore/lib/libcasa_casa.so` (compiled most probably with `/usr/bin/g++`, 4.8.5) contains a `casa::capitalize(std::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)` symbol, but a compilation with a newer compiler will require a `casa::capitalize(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)` symbol (mind the extra `__cxx11`).

In summary, you can't use the installation under `/usr/local/casacore` because you are using g++ >= 5.1, so maybe you can adjust the instructions to point to the `casacore/3.1.2` module installation as described above.

Reply:
Ahh, was unaware usr/local/casacore pointed to a different version. After merging with master this now depends on RapidJSON so I'll update that now